src/NodaTime.Test/Text/LocalDatePatternTest.cs

```diff
@@ -134,6 +134,8 @@ public class LocalDatePatternTest : PatternTestBase<LocalDate>
             // Invariant culture uses the crazy MM/dd/yyyy format. Blech.
             new Data(2011, 10, 20) { Pattern = "d", Text = "10/20/2011" },
             new Data(2011, 10, 20) { Pattern = "D", Text = "Thursday, 20 October 2011" },
+            // Year comes from the template
+            new Data(2000, 10, 20) { Pattern = "M", Text = "October 20" },
```

Comment:
I assume that a pattern of `"M"` before this is an error rather than falling back to the pattern `"%M"`?

Reply:
Yes.